app/src/components/shared/Switch.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import SwitchSelector from 'react-native-switch-selector';
+
+import { theme } from '../../theme';
+
+interface SwitchProps {
+  leftLabel: string;
+  rightLabel: string;
+  leftSideToggled: (arg: boolean) => void;
+}
+
+export const Switch: React.FC<SwitchProps> = (props) => {
+  const [leftBtnIsActive, setLeftBtnIsActive] = useState<boolean>(true);
```

Comment:
Takie zmienne warto zaczynać od `is` lub `has`, są wtedy bardziej czytelne, np: `isLeftButtonActive`